lib/os/assert.c

```diff
@@ -22,10 +22,16 @@
  *
  * @return N/A
  */
+#ifdef CONFIG_ASSERT_NO_FILE_INFO
+__weak void assert_post_action(void)
+#else
 __weak void assert_post_action(const char *file, unsigned int line)
+#endif
 {
+#ifndef CONFIG_ASSERT_NO_FILE_INFO
```

Comment:
Maybe we should just delete the file/line parameters since no configuration currently uses them.

Reply:
Isn't this handler supposed to be overridable by the application so external projects might be using it, I'm hesitent to remove it.

I'm referring to this:
```
 * System designers may wish to substitute this implementation to take other
 * actions, such as logging program counter, line number, debug information
 * to a persistent repository and/or rebooting the system.
```